tools/monorepo-utils/src/core/github/repo.ts

```diff
@@ -90,14 +90,21 @@ export const getRefFromGithubBranch = async (
 			}
 		` );
 
-	// @ts-ignore: The graphql query is typed, but the response is not.
-	return repository.ref.target.history.edges.shift().node.oid;
+	const target = repository.ref.target;
```

Comment:
The response is actually typed to some degree. There is only some ambiguity in the type of `oid`, which I consider a bug, but anyway, I think this is more typesafe than before because you could get back several objects for `target`. it could be Blob or Commit or (something else I forget), presence of `history` indicates its a `Commit`